connector/dialect.tdd

```diff
@@ -553,164 +285,19 @@
-      <formula>CORR(%1, %2)</formula>
-      <unagg-formula>NULL</unagg-formula>
-      <argument type='real' />
-      <argument type='real' />
-    </function>
-    <function group='aggregate' name='COUNT' return-type='int'>
-      <formula>COUNT(%1)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='bool' />
-    </function>
-    <function group='aggregate' name='COUNT' return-type='int'>
-      <formula>COUNT(%1)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='real' />
-    </function>
-    <function group='aggregate' name='COUNT' return-type='int'>
-      <formula>COUNT(%1)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='str' />
-    </function>
-    <function group='aggregate' name='COUNT' return-type='int'>
-      <formula>COUNT(%1)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='datetime' />
-    </function>
-    <function group='aggregate' name='COUNTD' return-type='int'>
-      <formula>COUNT(DISTINCT %1)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='bool' />
-    </function>
-    <function group='aggregate' name='COUNTD' return-type='int'>
-      <formula>COUNT(DISTINCT %1)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='real' />
-    </function>
-    <function group='aggregate' name='COUNTD' return-type='int'>
-      <formula>COUNT(DISTINCT %1)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='str' />
-    </function>
-    <function group='aggregate' name='COUNTD' return-type='int'>
-      <formula>COUNT(DISTINCT %1)</formula>
-      <unagg-formula>(case when %1 IS NULL THEN 0 ELSE 1 END)</unagg-formula>
-      <argument type='datetime' />
-    </function>
-    <function group='aggregate' name='COVAR' return-type='real'>
-      <formula>COVAR_SAMP(%1, %2)</formula>
-      <unagg-formula>NULL</unagg-formula>
-      <argument type='real' />
-      <argument type='real' />
-    </function>
-    <function group='aggregate' name='COVARP' return-type='real'>
-      <formula>COVAR_POP(%1, %2)</formula>
-      <unagg-formula>(CASE WHEN %1 IS NULL THEN NULL WHEN %2 IS NULL THEN NULL ELSE 0.0 END)</unagg-formula>
-      <argument type='real' />
-      <argument type='real' />
-    </function>
-    <function group='aggregate' name='MAX' return-type='bool'>
-      <formula>BOOL_OR(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
-      <argument type='bool' />
-    </function>
-    <function group='aggregate' name='MAX' return-type='real'>
-      <formula>MAX(%1)</formula>
-      <unagg-formula>%1</unagg-formula>
+    <function group='numeric;logical' name='ZN' return-type='real'>
```

Comment:
What is "ZN"?

Reply:
It's a function where the `null` values are converted to zero.  This one I've tested and get the expected output.

Reply:
Is this a function that tableau requires us to support? Is there a list of default functions they expect?

Reply:
This is a good design question. We should discuss this at stand-up tomorrow or at doc review on friday!

Reply:
As discussed at standup, I have removed the functions that are mainly implemented in the formula.  I have created SQL-794 to track the functions supported by Tableau that are not supported by MongoSQL.  We decided it is better to include only the functions that MongoSQL support as these will have been unit tested and would lead to less confusion for functions only in Tableau and not MongoSQL.